theme/static/js/hs_file_browser.js

```diff
@@ -131,22 +131,35 @@ function getFileTemplateInstance(file) {
     else {
         var title = '' + file.name + "&#13;Type: " + file.type + "&#13;Size: " +
             formatBytes(parseInt(file.size));
-    }
+    } 
+  const fileName = ellipsisFileName(file.name);
```

Comment:
I created a file with a long name:
![image](https://github.com/user-attachments/assets/cf2dc752-da59-4325-a0f6-f8a9364cade9)

If I rename the file using right click dialogue:
![image](https://github.com/user-attachments/assets/9e8bf9e6-02df-4c5d-ae5b-b9ef482ccc0b)
then I see the following which is not correct (should show the full name of the file):
![image](https://github.com/user-attachments/assets/4ff52af2-885d-4b87-adc7-db8b26c53355)
and if I edit the name of the file, I receive an error when trying to save the new name:
![image](https://github.com/user-attachments/assets/41555e58-fc80-4583-9dc7-0dc4a2bbab77)
from the POST request to http://localhost:8000/hsapi/_internal/data-store-rename-file-or-folder/